### DISCUSSION TITLE
hosting bottles from multiple `root_url`

### DISCUSSION THREAD

**Discussion by ipatch** [Homebrew/discussions#1273] [Tap maintenance and Homebrew development] [answered]
is it possible create a formula file that contains the `root_url` block, and have different bottles hosted on different TLDs?

i ask because i do some work with the homebrew-freecad tap, and i presently run macos mojave, and the main homebrew-freecad tap does not contain bottles for mojave, so i've been building bottles for the formula hosted on that tap. i set up a tag / release on my fork of the tap, and have been uploading the bottles as i build them. i'd like to submit a PR upstream to the main tap, but i won't bother if it's not possible to define multiple `root_url` within a formula file. curious if anyone has experience with such use case.

**Comment by SMillerDev** [comment, marked as answer]
It is not, but I guess they could just copy over the bottles.

**Reply by ipatch** [reply]
unfortunately i don't have write access to the `root_url` TLD. i'm giving my sales pitch on why we should be using github releases/tags to host the bottles.

i guess i'll have to wait and see how that goes.

thanks for the prompt reply 🙏.
